pkg/controllers/status/controller.go

```diff
@@ -108,7 +108,7 @@ func StartKubeFedStatusController(
 	if controllerConfig.MinimizeLatency {
 		controller.minimizeLatency()
 	}
-	klog.Infof(fmt.Sprintf("Starting status controller for %q", typeConfig.GetFederatedType().Kind))
+	controller.logger.WithValues("ftc-kind", typeConfig.GetFederatedType().Kind).Info("Starting status controller")
```

Comment:
```suggestion
	controller.logger.Info("Starting status controller")
```
ftc name should already have been added to the logger.

Reply:
done, pls take review.